nextflow_schema.json

```diff
@@ -206,14 +206,14 @@
         "annotation_bakta": {
             "title": "Annotation: BAKTA",
             "type": "object",
-            "description": "These parameters influence the annotation algorithm of Bacteria used by BAKTA.",
```

Comment:
For the annotation section above the BAKTA specific ones, the tutle should be `Annotation: General options` (like with BGC)